cardinal/plugins.py

```diff
@@ -184,6 +184,58 @@ def _create_plugin_instance(self, module, config=None):
 
         return instance
 
+    def _register_plugin_events(self, events):
+        """Registers events found in plugin
+
+        Will get all the events provided by _get_plugin_events though EventManager.
+        Callback IDs will be stored in registered_events so we can remove them on unload.
+        It is possible to have multiple methods as callback for single event.
+
+        Keyword arguments:
+            events - list of events to register
+
+        """
+
+        # Initialize variable to hold events callback IDs
+        registered_events = {}
+
+        # Loop though list of tuples
+        for event in events:
+            #Loop tought event names for each callback
+            for event_name in event[0]:
+                # Get callback ID from register_callback method
+                callback_id = self.cardinal.event_manager.register_callback(event_name, event[1])
+
+                # Append to list of callbacks for given event_name
+                if registered_events.has_key(event_name):
+                    registered_events[event_name].append(callback_id)
+                else:
+                    registered_events[event_name] = []
+                    registered_events[event_name].append(callback_id)
+
+        return registered_events
+
+    def _unregister_plugin_events(self, plugin):
```

Comment:
For consistency, maybe just take the list of `events` here, like `_register_plugin_events()`? Then probably return a list of any that failed to unregister.